hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -620,7 +622,8 @@ public static Map<String, HoodieData<HoodieRecord>> convertMetadataToRecords(Hoo
                                                                                HoodieMetadataConfig metadataConfig,
                                                                                List<MetadataPartitionType> enabledPartitionTypes,
                                                                                int bloomIndexParallelism,
-                                                                               Option<HoodieRecordType> recordTypeOpt) {
+                                                                               Option<HoodieRecordType> recordTypeOpt,
+                                                                               boolean enableOptimizeLogBlocksScan) {
```

Comment:
same as above, no need to set this flag.

Reply:
will remove